tools/dependencies-report/src/main/resources/licenseMapping.csv

```diff
@@ -0,0 +1,45 @@
+dependency,licenseOverride
+"gradle.plugin.com.github.jk1:gradle-license-report:0.7.1",Apache-2.0
+"webrick:1.3.1",BSD-2-Clause-FreeBSD
+"webhdfs:0.8.0",Apache-2.0
+"avl_tree:1.2.1",BSD-2-Clause-FreeBSD
+"filesize:0.0.4",MIT
+"filewatch:0.9.0",MIT
+"gelfd:0.2.0",Apache-2.0
+"gmetric:0.1.3",MIT
+"UNKNOWNgzip:1.0",UNKNOWN
+"UNKNOWNjls-grok:0.11.4",UNKNOWN
```

Comment:
We are blocked on this until @jordansissel adds a license to https://github.com/jordansissel/ruby-grok